mailpoet/lib/Migrations/Migration_20230419_080000.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Migrations;
+
+use MailPoet\Entities\NewsletterEntity;
+use MailPoet\Migrator\Migration;
+use MailPoet\Newsletter\NewslettersRepository;
+use MailPoet\Newsletter\Scheduler\PostNotificationScheduler;
+
+class Migration_20230419_080000 extends Migration {
+  /** @var NewslettersRepository */
+  private $newslettersRepository;
+
+  /** @var PostNotificationScheduler */
+  private $postNotificationScheduler;
+
+  public function run(): void {
+    $this->newslettersRepository = $this->container->get(NewslettersRepository::class);
+    $this->postNotificationScheduler = $this->container->get(PostNotificationScheduler::class);
+    $this->fixPostNotificationScheduleTime();
+  }
+
+  private function fixPostNotificationScheduleTime() {
```

Comment:
Could you please add a comment here describing what is this migration fixing? I think it would be helpful for the future.